flink-connectors/flink-connector-kafka/src/test/java/org/apache/flink/streaming/connectors/kafka/FlinkKafkaInternalProducerITCase.java

```diff
@@ -148,62 +153,78 @@ public void testResumeTransaction() throws Exception {
         deleteTestTopic(topicName);
     }
 
-    @Test(timeout = 30000L, expected = IllegalStateException.class)
-    public void testPartitionsForAfterClosed() {
+    @Test
+    @Timeout(30L)
```

Comment:
time unit should be specified

Reply:
By default it takes seconds, do we still need to specify?